sklearn/utils/estimator_checks.py

```diff
@@ -250,7 +251,12 @@ def check_estimator(Estimator):
     name = Estimator.__name__
     check_parameters_default_constructible(name, Estimator)
     for check in _yield_all_checks(name, Estimator):
-        check(name, Estimator)
+        try:
+            check(name, Estimator)
+        except SkipTest as message:
+            # the only SkipTest thrown currently results from not
+            # being able to import pandas.
+            warnings.warn(message, ImportWarning)
```

Comment:
``ImportWarning`` is ignored by default. I'd either use ``UserWarning`` or add a ``SkipTestWarning``. Feel free to add this to the PR, or change to ``UserWarning``. otherwise LGTM!